### DISCUSSION TITLE
Control without touchscreen?

### DISCUSSION THREAD

**Discussion by Mega1024** [opencardev/crankshaft#615] [Q&A]
I have an in-car screen that I'd like to use this with, but it is NOT a touchscreen. 

What sorts of physical controllers would work well to control 

Something that I could stick to my car's center console and operate by touch?

Is either of these devices likely to work?

https://www.amazon.com/dp/B08V95ZZLM/

https://www.amazon.com/dp/B0032Y0OH0/

edit: some additional clarification. The screen is "touch" (so there is no existing mechanical controller or anything) but I have no way of feeding that to the pi to control it. All I have is a way to feed video to it from the pi HDMI output (via an HDMI-to-NTSC adapter) so I need to use some sort of external add-on controller.

**Comment by techdev5521** [comment]
There's full support for binding keypresses (like from a keyboard) to operations in Crankshaft. You can see those optons in the settings menu. I'm unsure how the devices you linked to work -- if they're custom inputs that require a driver, simulate keypresses, can be configured etc.

Have you purchased any of these devices?
